lib/openxml/drawingml/properties/blip.rb

```diff
@@ -0,0 +1,28 @@
+module OpenXml
+  module DrawingML
+    module Properties
+      class Blip < OpenXml::Properties::ComplexProperty
+        include HasChildren
+        include ContainsProperties
```

Comment:
I'm OK with this style, but did you know you can also comma-separate included modules?

```ruby
include HasChildren, ContainsProperties
```

Reply:
👍